tests/table.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2017 AMIS Technologies
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package tests
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/onsi/ginkgo"
+)
+
+func CaseTable(description string, itBody interface{}, entries ...tableEntry) bool {
+	caseTable(description, itBody, entries, false, false)
+	return true
+}
+
+/*
+You can focus a table with `FCaseTable`.
+*/
+func FCaseTable(description string, itBody interface{}, entries ...tableEntry) bool {
+	caseTable(description, itBody, entries, false, true)
+	return true
+}
+
+/*
+You can mark a table as pending with `PCaseTable`.
+*/
+func PCaseTable(description string, itBody interface{}, entries ...tableEntry) bool {
+	caseTable(description, itBody, entries, true, false)
+	return true
+}
+
+/*
+You can mark a table as pending with `XCaseTable`.
+*/
+func XCaseTable(description string, itBody interface{}, entries ...tableEntry) bool {
+	caseTable(description, itBody, entries, true, false)
```

Comment:
Should the last parameter is true?
caseTable(description, itBody, entries, true, `false`)

Reply:
The last bool means focused. AFAIK, it's set to true only by FCase.